### PR TITLE
1.0.0

### PR DESCRIPTION
**Pull request by djperrefort**
This pull request adds official multi-site support and constitutes the first full package release. I would like to finish this PR before submitting reviewer comments on September 10th. As part of this, I'm requesting a review of the documentation from others within the WMWV group. (Key word here is requesting - I understand if that the first few weeks of the semester can be a very busy time).

Package documentation for the version implemented by this pull request can be found [here](https://mwvgroup.github.io/pwv_kpno/1.0.0/html/index.html). Note that since this pull request isn't merged yet, it is obviously not `pip` installable. If you wish to install and use this version of the package being reviewed you will have to run:

```bash
git clone https://github.com/mwvgroup/pwv_kpno/
cd pwv_kpno/
git checkout 1.0.0
python setup.py install --user
```

@burkfm and @MCilento can you please provide feedback on the "Installation and Setup" and "Accessing PWV Data" sections of the online docs?

@shuliu2017 and @jhand1993 can you please provide feedback on the "Modeling the Atmosphere" and "Modeling a Black Body" sections?

Specific questions:
    1. Is the documentation clear and easy to follow?
    2. Are there any portions of the documentation that you did not understand?
    3. Do you find the naming of various methods and arguments provide a clear intuition on what they do?
    4. Were you left with any questions that the documentation did not answer?

Reviews do not have to be extremely in-depth. Please make any comments by the end of **Saturday, September 8th**.

### CODE REVIEW

**Comment by MCilento** [comment]
Comments for 1. Installation and Setup:
--1.2 Package Setup
    >When I downloaded the SuomiNet data using the update_models function, it returned [2017,    2018]. In the documentation above the provided code to run the function, you mention that each release of pwv_kpno contains all necessary SuomiNet data from 2010 to the end of the previous year. So from that description I was confused by what the list [2017, 2018] meant. In installation documentation, link to the documentation for the update_models function to make the list clearer for users.  ***Also, after running update_models a second time, it returned the same list, which should not happen if I already ran it once. It should just return [2018].***
    >At the end of this section, you have the statement "Note that updating the locally available data requires the user to have permission to write and modify files within the package directory." I am just confused why as a user I need to know that. Is it in case someone wants to add more local data for their application? Then they know that they cannot do that without permission? 

--1.3 Running Tests
[Running Tests .txt](https://github.com/mwvgroup/pwv_kpno/files/2346921/Running.Tests.txt)
Above I copied and pasted the terminal window into a text file after I ran the test suites in this section. 4 tests failed when I expected them all to pass. 


Comments for 2. Accessing PWV Data:
--2.3 Measured PWV Data
    >In the paragraph starting with "Columns are named..." the second sentence starts weirdly. Do you mean to say "PWV measurements at each location..."

--2.4 Modeled PWV Data
    >In the 2.4.1 first example, when I run pwv_atm.modeled_pwv() I do not get the same table from 2010 onward. My table starts at 2015. I've been running all of the other code tests to see if my tables match and so far it was all good up until here. In the following file I copied my run statement and the resulting table: [2.4.1 example.txt](https://github.com/mwvgroup/pwv_kpno/files/2346949/2.4.1.example.txt). 

--2.5 PWV for a Given Date
    >In first sentence, minor spelling error: change "modeld" to "modeled"
    >Later on, in paragraph starting with "Interpolate from..." you have to "at"s right next to each other
    >In the returns, I think there should be a break after "site" where you move on to "The error..." Are they two separate returns?
    >In the 2.5.1 last example, there is nothing to confirm that everything was correctly downloaded, I simply generated variables. Maybe add a few more lines which show what the date and pwv variables should be after defining them?


Hopefully all of my comments make sense, if you have questions on what I mean just let me know!

**Comment by djperrefort** [comment]
@MCilento I've addressed your comments and fixed the bugs you've pointed out. Thanks again!!

**Comment by djperrefort** [comment]
@jhand1993 thanks for the comments, I’ll work on improving that section 👍🏻 As mentioned above I’d be particularly interested in your feedback for the questions regarding the online docs. They can be found [here](https://mwvgroup.github.io/pwv_kpno/1.0.0/html/index.html) did you have a chance to take a look at the “modeling the atmosphere” and “Modeling a black body” sections”?

File: pwv_kpno/_atm_model.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 __license__ = 'GPL V3'
 __email__ = 'djperrefort@pitt.com'
-__status__ = 'Development'
+__status__ = 'Release'
 
 
 def _calc_num_density_conversion():
```
[the remark this answers is not in the document]
Comment (djperrefort) [reply pwv_kpno/_atm_model.py]: This is done to provide developers with type hints why incorporating the package with their own code. I agree that the syntax is clumsy, but PEP 484 defines this as the recommended way to do type hinting for code supporting Python 2.7 and 3. I like the pure Python 3 syntax better:

    def create_pwv_atm_model(mod_lambda: np.ndarray, 
                             mod_cs: np.ndarray, 
                             out_lambda: np.ndarray) -> Table:

File: pwv_kpno/blackbody_with_atm.py [hunk 2]
```diff
@@ -58,7 +59,7 @@
 
 __license__ = 'GPL V3'
 __email__ = 'djperrefort@pitt.edu'
-__status__ = 'Development'
+__status__ = 'Release'
 
 
 def sed(temp, wavelengths, pwv):
```
[the remark this answers is not in the document]
Comment (djperrefort) [reply pwv_kpno/blackbody_with_atm.py]: Fixed - Thanks!

File: pwv_kpno/_update_pwv_model.py [hunk 3]
```diff
@@ -161,37 +162,52 @@ def _create_new_pwv_model(debug=False):
...
+                ' starting point.'
+            )
+
+        starting_year, ending_year = min(available_years), max(available_years)
+        all_years = set(range(starting_year, current_year + 1))
+        download_years = all_years - set(available_years)
+        download_years.add(ending_year)
+        years = sorted(download_years)
+
+    for year in years:
```
Comment (djperrefort) [inline pwv_kpno/_update_pwv_model.py]: I'll switch to using a generator.

**Comment by djperrefort** [comment]
The code for this PR is finished - I just need to wrap up some edits to the online docs

**Pull request merged**